src/relay/backend/vm/compiler.cc

```diff
@@ -1083,13 +1082,21 @@ IRModule VMCompiler::OptimizeModule(IRModule mod, const TargetsMap& targets_arg,
   pass_seqs.push_back(transform::InferType());
   pass_seqs.push_back(transform::LabelOps());
 
+  tec::TargetMap targets_;
+  tec::DeviceMap device_map;
+
+  targets_ = GetTecTargetMapFromTargetMap(targets);
+
+  pass_seqs.push_back(tec::LowerTEPass(targets_, device_map, "vm_mod", [this](Function func) {}));
```

Comment:
For others: This PR also needs to remove the direct calls to the TECompiler Lower and LowerShapeFunc which are a bit fiddly to change over. I believe Michalis is working on that.